src/components/MoneyRequestConfirmationList.js

```diff
@@ -499,6 +501,18 @@ function MoneyRequestConfirmationList(props) {
                             disabled={didConfirm || props.isReadOnly}
                         />
                     )}
+                    <View>
+                        <View style={[styles.flexRow, styles.mb4, styles.justifyContentBetween, styles.alignItemsCenter, styles.ml5, styles.mr8]}>
+                            <Text color={!props.iouIsBillable ? themeColors.textSupporting : undefined}>{translate('common.billable')}</Text>
+                            <Switch
+                                accessibilityLabel={translate('common.billable')}
+                                isOn={props.iouIsBillable}
+                                onToggle={(val) => {
+                                    props.onToggleBillable(val);
+                                }}
```

Comment:
We could simplify here:

```suggestion
                                onToggle={props.onToggleBillable}
```